bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/Connection.java

```diff
@@ -34,18 +27,16 @@
  */
 public class Connection {
 
-    public static final String REF_IDENTIFIER = "$";
-
-    private String ouputModuleId;
-    private String outputName;
-    private String inputName;
+    private final String ouputModuleId;
+    private final String outputName;
+    private final String inputName;
 
     /**
      * This constructor is responsible for creation of connections between modules in the rule.
      *
-     * @param inputName is an unique name of the {@code Input} in scope of the {@link Module}.
-     * @param ouputModuleId is an unique id of the {@code Module} in scope of the {@link Rule}.
-     * @param outputName is an unique name of the {@code Output} in scope of the {@link Module}.
+     * @param inputName is an unique name of the {@code Input} in scope of the {@link ModuleImpl}.
```

Comment:
Same here for ModelImpl vs. Model as above for Rule{,Impl}. I don't comment the other places....